tests/api/test_acl_range.py

```diff
@@ -0,0 +1,50 @@
+from pprint import pprint
+
+import pytest
+
+
+class TestSaiAclRange:
+    # object with no parents
+
+    @pytest.mark.dependency(scope='session')
+    def test_acl_range_create(self, npu):
+        commands = [
```

Comment:
In case we make it as a class variable and use `cleanup=True` for the remove command, the number of lines can be greatly reduced:
```
class TestSaiAclRange:
    # object with no parents
    commands = [
        {
            'name': 'acl_range_1',
            'op': 'create',
            'type': 'SAI_OBJECT_TYPE_ACL_RANGE',
            'attributes': [
                'SAI_ACL_RANGE_ATTR_TYPE',
                'SAI_ACL_RANGE_TYPE_L4_SRC_PORT_RANGE',
                'SAI_ACL_RANGE_ATTR_LIMIT',
                '10,20',
            ],
        }
    ]

    def test_acl_range_create(self, npu):
        results = [*npu.process_commands(self.commands)]
        print('======= SAI commands RETURN values create =======')
        pprint(results)
        assert all(results), 'Create error'

    def test_acl_range_remove(self, npu):
        results = [*npu.process_commands(self.commands, cleanup=True)]
        print('======= SAI commands RETURN values remove =======')
        pprint(results)
        assert all(
            [result == 'SAI_STATUS_SUCCESS' for result in results]
        ), 'Remove error'

```
The same is applicable to all other TCs.

Reply:
In case for some reason you still need to keep it separately, please clean up the `remove` command as follows:
```
    commands = [
        {
            'name': 'acl_range_1',
            'op': 'remove'
        }
    ]
```

Reply:
@selldinesh , could you share your plans to fix this? what approach you are going to choose? thanks

Reply:
this has been resolved